types/rxjs-exec/index.d.ts

```diff
@@ -0,0 +1,20 @@
+// Type definitions for rxjs-exec 0.4
+// Project: https://github.com/kssfilo/rxjs-exec
+// Definitions by: Meno Abels <https://github.com/mabels>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+/// <reference types="node" />
+
+import * as Rx from 'rxjs';
+
+export class Options {
+    binary?: boolean;
+    input?: string|Buffer;
+    stdin?: NodeJS.WritableStream;
+}
+
+export function exec(commandLine: string, options?: Options): Rx.Observable<Buffer>;
+export function execFilter(commandLine: string, options?: Options): Rx.Observable<Buffer>;
+export function mapExec(commandLine: string, options?: Options): Rx.Observable<Buffer>;
+export function mapExecFilter(commandLine: string, options?: Options): Rx.Observable<Buffer>;
+// declare export patch(rx: Rx.Observer);
```

Comment:
Hi,

i think the patch method is not ok for typescript, it monkey-patch rxjs module.
So you could apply this patch during runtime but rxjs will still not have the types.
So i decided to remove the patch function in the type definition.